### ISSUE TITLE
magic-point model

### ISSUE THREAD

**Issue by lililiiiiiiiiii** [rpautrat/SuperPoint#182]
hi,i want to train the superpoint on my own dataset? can you provide me the pretrained magic-point model to generate the label first?

since the traing is so slow

![image](https://user-images.githubusercontent.com/41133450/101648313-a5f75800-3a74-11eb-9222-d6f1a1269228.png)

**Comment by rpautrat** [comment]
Hi, the training shouldn't be slow. It took me ~2h to train MagicPoint on the synthetic shapes. But anyway, I sent you the pre-trained model per email.
